### ISSUE TITLE
ModuleNotFoundError: No module named 'avro.schema' when importing pyhwschema

### ISSUE THREAD

**Issue by swaapnika-guntaka** [kentore82/pyhwschema#4]
As mentioned in [https://github.com/kentore82/pyhwschema/issues/3](issue03) I rebuilt pyhwschema. But I still get an import error for avro. When I just import avro it works fine but pyhwschema throws error.

```
Collecting git+https://github.com/kentore82/pyhwschema
  Cloning https://github.com/kentore82/pyhwschema to /tmp/pip-req-build-626ehk08
Unpacking objects: 100% (75/75), done.
Requirement already satisfied (use --upgrade to upgrade): pyhwschema==0.0.1 from git+https://github.com/kentore82/pyhwschema in /remote/vgrnd104/guntaka/anaconda3/lib/python3.6/site-packages
Requirement already satisfied: requests in /remote/vgrnd104/guntaka/anaconda3/lib/python3.6/site-packages (from pyhwschema==0.0.1) (2.18.4)
Requirement already satisfied: avro-python3 in /remote/vgrnd104/guntaka/anaconda3/lib/python3.6/site-packages (from pyhwschema==0.0.1) (1.8.2)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /remote/vgrnd104/guntaka/anaconda3/lib/python3.6/site-packages (from requests->pyhwschema==0.0.1) (3.0.4)
Requirement already satisfied: idna<2.7,>=2.5 in /remote/vgrnd104/guntaka/anaconda3/lib/python3.6/site-packages (from requests->pyhwschema==0.0.1) (2.6)
Requirement already satisfied: urllib3<1.23,>=1.21.1 in /remote/vgrnd104/guntaka/anaconda3/lib/python3.6/site-packages (from requests->pyhwschema==0.0.1) (1.22)
Requirement already satisfied: certifi>=2017.4.17 in /remote/vgrnd104/guntaka/anaconda3/lib/python3.6/site-packages (from requests->pyhwschema==0.0.1) (2018.4.16)
Building wheels for collected packages: pyhwschema
  Running setup.py bdist_wheel for pyhwschema ... done
  Stored in directory: /tmp/pip-ephem-wheel-cache-qn7zbpst/wheels/d0/de/c8/6e681c5a44c747a9c306fcc17792850ea565089ff6730b6e8f
Successfully built pyhwschema
```

```
% python
Python 3.6.5 |Anaconda, Inc.| (default, Apr 29 2018, 16:14:56) 
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyhwschema
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "anaconda3/lib/python3.6/site-packages/pyhwschema/__init__.py", line 1, in <module>
    from .schema import HwSchema
  File "anaconda3/lib/python3.6/site-packages/pyhwschema/schema.py", line 3, in <module>
    from avro.schema import parse
ModuleNotFoundError: No module named 'avro.schema'

```

**Comment by kentore82** [comment]
fixed in avro==1.10.0

**Comment by fabianheredia** [comment]
1.10.0 don't work
